recipe/meta.yaml

```diff
@@ -19,17 +19,26 @@ requirements:
   build:
     - {{ compiler('c') }}
     - make
+    - cfdm
+    - cftime
+    - cfunits
+    - netcdf4
+    - netcdf-flattener
+    - numpy
+    - psutil
+    - packaging
+    - pyparsing
   host:
     - python
     - pip
   run:
     - python
-    - cfdm >=1.9.0.0,<1.9.1.0
-    - cftime >=1.5.0
-    - cfunits >=3.3.3
-    - netcdf4 >=1.5.4
-    - numpy >=1.15
-    - psutil >=0.6.0
```

Comment:
This library is heavily pinned upstream: https://github.com/NCAS-CMS/cf-python/blob/v3.13.0/requirements.txt

Reply:
I've had numerous problems with conda forge packages which are heavily pinned in the run requirements, but when versions are updated these pins are not updated, and the released conda package ends up either uninstallable, or unuseable, because of those pins. If we can avoid having to use them, I think it would be better for package maintainability.

Reply:
Please open an issue upstream about that. I do update packages with grayskull only prevent stale pins, so you are seeing issues mostly where automerge is used without grayskull.

A lose pin like that without full tests is prone to break and not future proof. Upstream has tons of tests to ensure these intervals work as expected.

Reply:
Ahh, I wasn't aware of grayskull - that tool does look very useful for pip based packages. I'll make a note and suggest it's use next time I have this problem.

I've reverted my changes here.

Reply:
We have an experimental functionality to update automatically with grayskull using the bot. When that is out most of the headaches you mentioned should be fixed (we hope).